conda-forge.yml

```
@@ -1,4 +1,6 @@
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Not sure why the migrator added ppc, it's not supported, and running into https://github.com/conda-forge/pandoc-feedstock/issues/133
```suggestion
  linux_aarch64: linux_64
```